### PR TITLE
[PE-4855] Backstop test for button vertical group

### PR DESCRIPTION
**Pull request by sabah1910**
@jllr @mattnickles @janibolkvadze https://ctl.atlassian.net/browse/FSPE-4855

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://090cac6fcff1b4fb4d9b822a835b772d3b5db4d2-chi-pr.enterprise.ctl.io" target="_blank"> https://090cac6fcff1b4fb4d9b822a835b772d3b5db4d2-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1199/ for more details.

File: src/custom-elements/src/components/date/date.tsx [hunk 1]
```diff
@@ -150,7 +150,7 @@ export class Date {
   };
 
   private _initCalendarViewModel(): void {
-    dayjs.locale(this.locale);
+    // dayjs.locale(this.locale);
```
Comment (janibolkvadze) [inline src/custom-elements/src/components/date/date.tsx:153]: @sabah1910 Please uncomment this line and Rebase from master

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://b29e89b9ae071da75ef51b81042845343f6529f7-chi-pr.enterprise.ctl.io" target="_blank"> https://b29e89b9ae071da75ef51b81042845343f6529f7-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1229/ for more details.

File: src/custom-elements/src/components/date/date.tsx (hunk 1 above)
Comment (sabah1910) [reply src/custom-elements/src/components/date/date.tsx]: @janibolkvadze Done

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://a633eb69388ebd37df707c65b49e32c52d655956-chi-pr.enterprise.ctl.io" target="_blank"> https://a633eb69388ebd37df707c65b49e32c52d655956-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1240/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-z6er53, https://ops.elasticbox.com/services/instances/i-w42bl7, https://ops.elasticbox.com/services/instances/i-2gt47d

**Pull request merged**